ShuttleTrackeriOS/ViewController.swift

```diff
@@ -42,40 +37,19 @@ class ViewController: UIViewController, MGLMapViewDelegate {
     @IBAction func toggleRoutes(_ sender: UISegmentedControl) {
         switch sender.selectedSegmentIndex {
         case 0:
```

Comment:
what about just late night?

Is there a way to make the bar dynamic based on how many routes are being shown instead of hard coding the 3 cases?

Reply:
Also what about the holiday/inclement weather schedule?

Reply:
I'm not sure about the UI for late night, so I don't know how I should display it. Right now my idea is to enable/disable late night based on time. 
Oh probably at a certain time add/remove a new segment called "late night only"?

What do you mean by "based on how many routes being shown"?